### PR TITLE
Improve demo data

### PR DESCRIPTION
**Pull request by macobo**
This grabs https://github.com/PostHog/posthog/pull/2629 and adapts it to work with new data generation and gets rid of a lot of duplication.

- Unify clickhouse and pg demo data creation
- Adapt yakkos demo data generation to normal demo
- Split demo.py into subfolder
- Add original, web data import
- Improve wording
- dont generate dashboards

## Changes

*Please describe.*  
*If this affects the frontend, include screenshots.*  

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests

### CODE REVIEW

File: cypress/integration/person.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 describe('Person Visualization Check', () => {
     beforeEach(() => {
         cy.get('[data-attr=menu-item-people]').click() // TODO: Adjust when releasing navigation-1775
+        cy.get('[data-attr=menu-item-persons].ant-menu-item-selected').should('be.visible')
```
Comment (paolodamico) [inline cypress/integration/person.js:4]: Context on these changes? Seem unrelated to the PR

File: posthog/test/test_demo.py [hunk 2]
```diff
@@ -6,13 +8,15 @@ class TestDemo(BaseTest):
...
 
     def test_create_demo_data(self):
+        random.seed(900)
+
         self.client.get("/demo")
         demo_team = Team.objects.get(name__icontains="demo")
-        self.assertEqual(Event.objects.count(), 192)
-        self.assertEqual(Person.objects.count(), 100)
-        self.assertEqual(Action.objects.count(), 3)
+        self.assertEqual(Event.objects.count(), 965)
```
Comment (paolodamico) [inline posthog/test/test_demo.py:15]: Maybe out of the scope of this PR, but think it's quite hard to verify these numbers, not sure if this is really helpful to test expected behaviour

File: cypress/integration/person.js (hunk 1 above)
Comment (macobo) [reply cypress/integration/person.js:4]: Demo data changed making changes on how navigation works needed. 

In this case, there are tabs on the page we start from so we need to wait for the new page to load first.

Ideally we should detangle e2e tests from demo data but that's a longer spiel.

**Review by paolodamico — APPROVED** [review]
bottom line, what we have, lgtm!

* The failing test looks the same flaky test that is always misbehaving, rerunning test suite.
* What we have works great so far! Though I do think that there are some things that we're missing, that are necessary for this great new feature to be truly valuable. Feel free to spin off as a separate issue (and we can even move it to the next release, but I do think we need to focus on this).
    * The dashboards seem quite plain and not really showcasing all the capability of PostHog (e.g. funnels, active usage vs. volume, ...).
    * Related to the above, I think we need to think from use cases first. For instance, for the Web Analytics dashboard, we would expect to see all the information we get on GA (e.g. where are my users coming from [referrer breakdown], visitor count, session duration information, locations [future improvement].
    * The generated data doesn't seem very useful to see retention numbers.
    * Another use case I think we're missing, particularly for websites and marketing people, ad campaign stuff. Have a dashboard that displays sample metrics of different paid ad campaigns and their performance (UTM tags).
    * Sample session recordings.

@yakkomajuri as you've done a ton of demos & created the playground, you may have more insight on this!

File: posthog/test/test_demo.py (hunk 2 above)
Comment (macobo) [reply posthog/test/test_demo.py:15]: Agreed. We need to verify _some_ data exists but the numbers are quite arbitrary. Will think of this next time I change the test.
